### PR TITLE
Better describe changes in CHANGELOG.md

### PR DESCRIPTION
**Pull request by y-guyon**

### CODE REVIEW

**Review by vrabaud — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ return values of avifImageCopy() and avifImageAllocatePlanes().
   and avifRGBImage::avoidLibYUV to AVIF_TRUE.
 * avifRGBImage::chromaUpsampling now only applies to conversions that need
   upsampling chroma from 4:2:0 or 4:2:2 and has no impact on the use of libyuv.
+  Set avifRGBImage::avoidLibYUV accordingly for that.
```
Comment (wantehchang) [inline CHANGELOG.md:30]: Nit: "for that" is not clear. How about "to control the use of libyuv"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -39,7 +40,9 @@ return values of avifImageCopy() and avifImageAllocatePlanes().
 * Add imageDimensionLimit field to avifDecoder struct
 * Add autoTiling field to avifEncoder struct
 * Add AVIF_CHROMA_DOWNSAMPLING_SHARP_YUV value to avifChromaDownsampling enum
-* Add avifImageExtractExifOrientationToIrotImir()
+* Add avifImageExtractExifOrientationToIrotImir() which parses the Exif payload
```
Comment (wantehchang) [inline CHANGELOG.md:43]: Nit: payload => metadata

**Review by wantehchang — COMMENTED** [review]
LGTM. Thanks. I suggest two edits.

File: CHANGELOG.md (hunk 1 above)
Comment (y-guyon) [reply CHANGELOG.md:30]: Done in https://github.com/AOMediaCodec/libavif/pull/1131

File: CHANGELOG.md (hunk 2 above)
Comment (y-guyon) [reply CHANGELOG.md:43]: Done in https://github.com/AOMediaCodec/libavif/pull/1131

**Pull request merged**